docs/user/azure/install.md

```diff
@@ -18,6 +18,7 @@ Step 3: Download the Installer.
 ? azure tenant id xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
 ? azure service principal client id xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
 ? azure service principal client secret xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+INFO Saving user credentials to "/home/demo/.azure/osServicePrincipal.json"
```

Comment:
User home directory should be consistent. For the SSH Public Key step it is `/home/user_id/` here it is `/home/demo/`